scripts/extract_translation_templates.py

```diff
@@ -90,11 +91,35 @@ def check_schema_templates(source_dir, target_dir):
 
 
 if __name__ == "__main__":
+
+    page = requests.get("https://github.com/ONSdigital/eq-translations/commit/master/")
+
+
+    def get_ref():
+        f = open("Pipfile.lock", "r")
+        file = f.readlines()
+        for i, line in enumerate(file):
+            if '"eq-translations"' in line:
+                ref = re.compile(r"\w{40}")
+                mo = ref.search(file[i + 3])
+                return mo.group()
+        print("eq-translations not in Pipfile.lock")
+
+
+    if get_ref() in page.text:
+        print("eq-translations up to date")
+
+    else:
+        print('Newer version of eq-translations available, use "pipenv update"')
```

Comment:
We should use `logger` not `print` and this could be `logger.error`

Reply:
Used logger.error! Thanks!👍